src/types/api/rpc.ts

```diff
@@ -143,7 +143,7 @@ export namespace RPC {
     starknet_getNonce: {
       QUERY: never;
       REQUEST: any[];
-      RESPONSE: string;
+      RESPONSE: any;
```

Comment:
Should this be now also BigNumberish like
https://github.com/0xs34n/starknet.js/pull/318/files#diff-6ae3f9177722bfbaef720fe10fadc424e40d7895481d9a7c09aac78cccf3b17fR109